linode/manager
# chore: Repo Clean Up

## Description 📝

- This is an extension of https://github.com/linode/manager/pull/8907
- Does more clean up in the repo by deleting old unused code
- Removes some js files that were somehow generated in https://github.com/linode/manager/pull/8911

## How to test 🧪

- Confirm components that were touched still function as expected
  - Mainly Linode Create
